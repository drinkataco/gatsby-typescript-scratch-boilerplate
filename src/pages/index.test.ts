import IndexPage from '.';

describe('index page', () => {
  it('renders page', () => {
    expect.assertions(1);

    expect(IndexPage()).toMatchSnapshot();
  });
});
