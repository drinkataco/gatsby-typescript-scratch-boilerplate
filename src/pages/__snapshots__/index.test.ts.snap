// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page renders page 1`] = `
<MainLayout>
  <h1>
    Homepage
  </h1>
  <p>
    Welcome to your new Gatsby site.
  </p>
  <p>
    This is a template for you to build your new site with. This index page is static. But click around the navigation to see content rendered from Markdown!
  </p>
</MainLayout>
`;
